src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -2335,6 +2350,18 @@ export class SavedObjectsRepository {
       throw SavedObjectsErrorHelpers.createBadRequestError(error.message);
     }
   }
+
+  /** This is used when objects are created. */
+  private validateOriginId(type: string, objectOrOptions: { originId?: string }) {
```

Comment:
What other use cases would we have for `objectOrOptions`? ATM, we're only using `originId` or am I missing something?